CorsixTH/Lua/objects/door.lua

```diff
@@ -166,6 +166,8 @@ function Door:checkForDeadlock()
     for _, action in ipairs(self.reserved_for.action_queue) do
       if action.name == "queue" then
         if action.queue ~= self.queue or self.queue[1] ~= self.reserved_for then
+          print("Warning: Trying to resolve door deadlock at ("
```

Comment:
While I myself am a fan of ``print()`` I don't like it in the end code. A better convention perhaps (albeit achieves the same thing) is switching to ``self.world:gameLog()`` instead.

Reply:
done